pkg/adapter/apiserver/delegate_test.go

```diff
@@ -18,43 +18,43 @@ package apiserver
 import (
 	"testing"
 
-	sourcesv1alpha1 "knative.dev/eventing/pkg/apis/sources/v1alpha1"
+	sourcesv1beta1 "knative.dev/eventing/pkg/apis/sources/v1beta1"
 )
 
 func TestResourceAddEvent(t *testing.T) {
 	d, ce := makeResourceAndTestingClient()
 	d.Add(simplePod("unit", "test"))
-	validateSent(t, ce, sourcesv1alpha1.ApiServerSourceAddEventType)
+	validateSent(t, ce, sourcesv1beta1.ApiServerSourceAddEventType)
```

Comment:
it's unfortunate that we have coupled the type to the versions, as all versions use the same type. Would have made more sense to have this constants un one package above, in the register.go for example. 
But not need to change it now, I think we can then do cleanups after this

Reply:
Follow up issues created: https://github.com/knative/eventing/issues/3628